components/benchmarks.vue

```diff
@@ -50,11 +50,7 @@
                 Platforms
               </v-expansion-panel-header>
               <v-expansion-panel-content class="text--secondary">
-                Simulations were run using the following platforms and hardware:
-                <ul>
-                  <li>CUDA platform on a variety of Volta (Titan V, V100), Turing (RTX 2080, RTX 2080 Ti), and Ampere (Quadro RTX 6000, A10, A40, A100) GPUs.</li>
-                  <li>CPU platform on a 4 core 4.3 GHz Intel Core i7-7740X CPU.</li>
-                </ul>
+                Simulations were run using the CUDA platform on a variety of Ampere (RTX 3090, A100) and Hopper (RTX 4080, H100) GPUs.
```

Comment:
The architecture of the H100 is based on NVIDIA Hopper, while the RTX 4080 is based on NVIDIA Ada Lovelace, and these two architectures are fundamentally distinct from each other. These are their whitepapers: [Hopper](https://resources.nvidia.com/en-us-tensor-core), [Ada Lovelace](https://images.nvidia.com/aem-dam/Solutions/geforce/ada/nvidia-ada-gpu-architecture.pdf)

Furthermore, even though both the A100 and RTX 3090 are referred to as Ampere, their architectures are also fundamentally different from each other.